addons/im_livechat/static/tests/embed/livechat_service.test.js

```diff
@@ -88,13 +88,22 @@ test("Only necessary requests are made when creating a new chat", async () => {
     await start({ authenticateAs: false });
     await contains(".o-livechat-LivechatButton");
     await waitForSteps([
+        // lazy_session_info is called because mail/core/web is loaded in test bundle
+        `/web/dataset/call_kw/ir.http/lazy_session_info - ${JSON.stringify({
+            model: "ir.http",
+            method: "lazy_session_info",
+            args: [[]],
+            kwargs: {
+                context: {
+                    lang: "en",
+                    tz: "taht",
+                    uid: serverState.userId,
+                    allowed_company_ids: [1],
+                },
+            },
+        })}`,
```

Comment:
Same remark: we could filter out `lazy_session_info` from the `asyncStep`. The awaiting of `init_livechat` seems good enough for technical reasons.